contribs/gmf/src/directives/layertree.js

```diff
@@ -193,6 +193,8 @@ gmf.LayertreeController.prototype.getLayer = function(node) {
   } else {
     layer = this.layerHelper_.createBasicWMSLayer(layerURL, layerName);
   }
+  // FIXME
```

Comment:
Yes.  I left this note in case the layer creation had been moved at some place else but that doesn't seem to be the case yet so I'll just remove the flag.